app/Jobs/SendSubscribedNotificationsJob.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Mail\CheckInNotificationMail;
+use App\Models\PhoneNumber;
+use App\Models\Subscription;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Mail;
+
+class SendSubscribedNotificationsJob implements ShouldQueue, ShouldBeUnique
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    public function __construct(
+        public PhoneNumber $phone_number,
+    )
+    {
+    }
+
+    public function uniqueId(): string
+    {
+        return $this->phone_number->id;
+    }
+
+    public function handle(): void
+    {
+        $check_ins = $this->phone_number->not_notified_check_ins()->get();
+
+        if ($check_ins->count()) {
+            $this->phone_number->subscriptions()
+                ->each(function (Subscription $subscription) use ($check_ins) {
+                    Mail::to($subscription->user->email)->send(
```

Comment:
`->queue()` here instead of `->send()` ?

Reply:
Likely OK to do send here since we are in a queued job already. I don't think, for now at least, that there will be huge numbers of subscribers.